lib/dirac_mobius.cpp

```diff
@@ -125,17 +113,17 @@ namespace quda {
     ApplyDslash5(out, in, x, mass, m5, b_5, c_5, k, dagger, DSLASH5_MOBIUS);
 
     long long Ls = in.X(4);
-    long long bulk = (Ls-2)*(in.Volume()/Ls);
-    long long wall = 2*in.Volume()/Ls;
-    flops += (96LL)*(long long)in.Volume() + 96LL*bulk + 120LL*wall;
+    long long bulk = (Ls - 2) * (in.Volume() / Ls);
+    long long wall = 2 * in.Volume() / Ls;
+    flops += (96LL) * (long long)in.Volume() + 96LL * bulk + 120LL * wall;
   }
 
   void DiracMobius::M(ColorSpinorField &out, const ColorSpinorField &in) const
   {
     checkFullSpinor(out, in);
 
     // FIXME broken for variable coefficients
-    double kappa_b = 0.5 / (b_5[0].real() * (4.0 + m5) + 1.0);
+    double mobius_kappa_b = 0.5 / (b_5[0].real() * (4.0 + m5) + 1.0);
```

Comment:
Re:my request for checking for variable coefficients in the constructor, add a reference to that check here.